openapi/src/openapi-src-v2.json

```diff
@@ -274,6 +280,176 @@
+                                    "$ref": "json-schemas/api/v2_questionnaires_metadata/get/res/200.json"
+                                },
+                                "example": {
+                                    "data": [
+                                        {
+                                            "type": "metadata",
+                                            "id": "285cb104-0c15-4a9c-9840-cb1007f098fb",
+                                            "attributes": {
+                                                "questionnaire-id": "285cb104-0c15-4a9c-9840-cb1007f098fb",
+                                                "questionnaire-document-version": "4.1.0",
+                                                "created": "2022-01-18T04:05:06Z",
+                                                "modified": "2022-01-18T05:05:06Z",
+                                                "submission-status": "IN_PROGRESS",
+                                                "expires": "2023-01-24T05:05:06Z"
+                                            }
+                                        }
+                                    ]
+                                }
+                            }
+                        }
+                    },
+                    "400": {
+                        "$ref": "#/components/responses/BadRequest"
+                    },
+                    "401": {
+                        "$ref": "#/components/responses/UnauthorizedError"
+                    },
+                    "403": {
+                        "$ref": "#/components/responses/Forbidden"
+                    },
+                    "404": {
+                        "$ref": "#/components/responses/NotFound"
+                    }
+                }
+            }
+        },
+        "/questionnaires/{questionnaireId}/metadata": {
+            "get": {
+                "tags": ["Metadata"],
+                "summary": "Get the metadata associated with a specific questionnaires",
+                "description": "Gets a single questionnaire's metadata. The results are filtered by owner.",
+                "operationId": "getQuestionnaireMetadata",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/questionnaireId"
+                    },
+                    {
+                        "$ref": "#/components/parameters/onBehalfOf"
+                    },
+                    {
+                        "$ref": "#/components/parameters/apiVersion"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success",
+                        "content": {
+                            "application/vnd.api+json": {
+                                "schema": {
+                                    "$ref": "json-schemas/api/v2_questionnaires_metadata/get/res/200.json"
```

Comment:
Both metadata endpoints are using `json-schemas/api/v2_questionnaires_metadata/get/res/200.json` to define the shape. Is this right? One is a single resource and one is a collection (potentially of size:1), but they should be represented as such.

Reply:
It was not right. I've updated this.